data/transactions/logic/opcodes.go

```diff
@@ -513,9 +513,9 @@ var OpSpecs = []OpSpec{
 	{0x04, "ed25519verify", opEd25519Verify, proto("b63:T"), 1, costly(1900).only(ModeSig)},
 	{0x04, "ed25519verify", opEd25519Verify, proto("b63:T"), 5, costly(1900)},
 
-	{0x05, "ecdsa_verify", opEcdsaVerify, proto("3bbbb:T"), 5, costByField("v", &EcdsaCurves, ecdsaVerifyCosts)},
-	{0x06, "ecdsa_pk_decompress", opEcdsaPkDecompress, proto("b:bb"), 5, costByField("v", &EcdsaCurves, ecdsaDecompressCosts)},
-	{0x07, "ecdsa_pk_recover", opEcdsaPkRecover, proto("3i33:bb"), 5, field("v", &EcdsaCurves).costs(2000)},
+	{0x05, "ecdsa_verify", opEcdsaVerify, proto("33333:T"), 5, costByField("v", &EcdsaCurves, ecdsaVerifyCosts)},
```

Comment:
Good callout.  If we did support such a curve we could either remove the annotation (since it's only assembly behavior) or add support for different types depending on the immediate. We have a rudimentary form of that for the return types from `txn`. It's not in the proto, but we do pay attention to the different return types for, say `Sender` and `Fee`.